meta-freescale-mbl/recipes-bsp/u-boot/u-boot-fslc_%.bbappend

```diff
@@ -28,6 +28,11 @@ do_compile_append_imx7s-warp-mbl() {
 
 DCD_FILE_PATH_imx7s-warp-mbl = "${B}"
 DCD_FILE_PATH_imx7d-pico-mbl = "${WORKDIR}"
+DCD_FILE_PATH_imx6ul-pico-mbl = "${B}"
+
+do_deploy_prepend_imx6ul-pico-mbl() {
```

Comment:
`do_deploy_prepend_imx6ul-pico-mbl` is not needed because `do_deploy_append` is already copying the `u-boot-dtb.cfgout` to de `${DEPLOYDIR}`

Reply:
I need that though, doesn't build otherwise

Reply:
Any idea why for `pico-pi-imx6ul_defconfig` it doesn't generate a `u-boot-dtb.cfgout`?

Reply:
I think that is because the `pico-pi-imx6ul` doesn't have a device-tree file and the `u-boot-dtb.cfgout` is not being generated.

I believe we can have this tweak and when we add the device tree for it we remove it.